handler/src/test/java/io/netty/handler/ssl/OpenSslKeyMaterialProviderTest.java

```diff
@@ -138,15 +138,14 @@ public PrivateKey getPrivateKey(String alias) {
 
     @Test
     public void testChooseOpenSslPrivateKeyMaterial() throws Exception {
-        PrivateKey privateKey;
-        try (InputStream keyInputStream = getClass().getResourceAsStream("localhost_server.key")) {
-            privateKey = SslContext.toPrivateKey(keyInputStream, null);
-            assertNotNull(privateKey);
-            assertEquals("PKCS#8", privateKey.getFormat());
-        }
-
+        PrivateKey privateKey = SslContext.toPrivateKey(
+                getClass().getResourceAsStream("localhost_server.key"),
```

Comment:
Class loader resource streams need to be closed AFAIK, and `toPrivateKey` isn't closing the stream its given.